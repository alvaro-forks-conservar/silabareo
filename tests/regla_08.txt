obvio
abdicar
subprefecto
obtener
subfebril
obcecado
abyecto
subgénero
chibcha
submarino
abnegar
anécdota
acto
becqueriano
acción
picnic
subgobernador
subcampeón
bricbarba
pecblenda
macferlán