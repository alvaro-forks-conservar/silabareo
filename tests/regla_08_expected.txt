ob-vio
ab-di-car
sub-pre-fec-to
ob-te-ner
sub-fe-bril
ob-ce-ca-do
ab-yec-to
sub-gé-ne-ro
chib-cha
sub-ma-ri-no
ab-ne-gar
a-néc-do-ta
ac-to
bec-que-ria-no
ac-ción
pic-nic
sub-go-ber-na-dor
sub-cam-pe-ón
bric-bar-ba
pec-blen-da
mac-fer-lán